themes/default/content/what-is/what-is-configuration-management.md

```diff
@@ -0,0 +1,73 @@
+---
+title: What is Configuration Management?
+allow_long_title: true
+meta_desc: |
+    Learn about what configuration management is and why configuration management is instrumental in maintaining the health and consistency of software systems.
+type: what-is
+page_title: What is Configuration Management?
+---
+
+## What is configuration management?
+
+Configuration management is a systems engineering process for handling changes to a system's functionality, performance, and attributes in a way that preserves the system's integrity. Put more simply, it's the process of maintaining and controlling the desired state of a system to ensure it behaves as expected under various conditions.
+
+Although configuration management originated outside the technology space, the term is now frequently used to refer to the process of managing and maintaining the desired state of various IT elements---such as networks, servers, operating systems, clusters of servers, or pieces of software. As such, it's not uncommon to see the usage of terms like _network configuration management_, _software configuration management_, or _server configuration management_ to qualify the scope of a configuration management tool or project. Other common terms used include _server orchestration_ and _IT automation_.
+
+Configuration management encompasses three broad aspects of managing and maintaining the desired state of the systems:
+
+1. **Identification**: Configuration management helps identify the systems that need to be reconfigured, patched, or updated in order to align with the desired state, as well as identifies the specific changes that need to be made to those systems.
+2. **Remediation**: Configuration management helps remediate the changes necessary to bring systems into alignment with the desired state. This could include applying software or security patches, installing (or uninstalling) specific software packages, or changing operating system configuration parameters.
+3. **Automation**: Configuration management helps automate both of the previous aspects (identification and remediation).
+
+## What are some common configuration management tools?
+
+There are a number of configuration management tools on the market, but here are a few of the most commonly mentioned tools:
```

Comment:
Thoughts on linking to each of the tools below?

Reply:
Seems reasonable, done